lib/watchErrorHandler.js

```diff
@@ -0,0 +1,7 @@
+module.exports = (message, status) =>  {
+  if(status === 0) {
+    console.log(message);
+  } else {
+    console.error( message.stack ? message.stack : String(message));
```

Comment:
@ryanerato The `String(message)` stuff is consistent with how the [hard exit handler](https://github.com/3dna/bundbi/blob/master/lib/exit.js) is written. What do you think of rolling forward with that, for consistency? (I've pushed up the `||` tweak though.)